### PR TITLE
Add instructions link to the first step of adding a safe

### LINKED ISSUE #4082: Add instructions card link on first step
### Description

During the live QA session we noticed that we don't provide any instructions at the first step of the Add a Safe process, resulting in the user being confused as to what the feature is used for and what will be required to set it up.

This adds a new call out section that encourages the user to review the instructions of what they will need during the process and instructions on how to set it up.

The `Set up instructions` link should go to - https://colony.gitbook.io/colony/advanced-features/safe-control-gnosis-safe/adding-a-safe

### Design

[Figma link - New step 1 callout section](https://www.figma.com/file/LlVmeMQxHVA04evfrgZyuN/Safe-Control?node-id=8138%3A24466&t=1MytesFhwrIVww6u-1)

first-step-instructions

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
FireShot Capture 763 - Actions - Colony - wonker - localhost

Resolves #4082

### CODE REVIEW

**Review by Julianahlk — APPROVED** [review]
LGTM 👍

**Review by chinins — APPROVED** [review]
Good to go!

File: src/modules/dashboard/components/Dialogs/AddExistingSafeDialog/CheckSafe.tsx [hunk 1]
```diff
@@ -51,6 +53,14 @@ const MSG = defineMessages({
     id: 'dashboard.AddExistingSafeDialog.CheckSafe.safe',
     defaultMessage: 'Safe',
   },
+  callout: {
+    id: 'dashboard.AddExistingSafeDialog.CheckSafe.callout',
+    defaultMessage: '<span>Important!</span>  Read instuctions before starting',
```
Comment (arrenv) [inline src/modules/dashboard/components/Dialogs/AddExistingSafeDialog/CheckSafe.tsx:58]: I blame myself, but there is a typo here in "instuctions", it should be "Important! Read instructions before starting"

**Review by arrenv — CHANGES_REQUESTED** [review]
Just a small typo to fix. 

EDIT: Sorry to do this but Karol made some design updates after the design review, they should be minor, you can find them here: https://www.figma.com/file/LlVmeMQxHVA04evfrgZyuN/Safe-Control?node-id=8187%3A25003&t=0tPMrXoN383cJxhi-1

Screenshot 2022-12-02 130318

File: src/modules/dashboard/components/Dialogs/AddExistingSafeDialog/CheckSafe.tsx [hunk 2]
```diff
@@ -51,6 +54,18 @@ const MSG = defineMessages({
     id: 'dashboard.AddExistingSafeDialog.CheckSafe.safe',
     defaultMessage: 'Safe',
   },
+  callout: {
+    id: 'dashboard.AddExistingSafeDialog.CheckSafe.callout',
+    defaultMessage: '<span>Important!</span>  Read the instuctions first.',
```
Comment (arrenv) [inline src/modules/dashboard/components/Dialogs/AddExistingSafeDialog/CheckSafe.tsx:59]: @ArmandoGraterol We have the typo here still "instuctions: should be "instructions"

File: src/modules/dashboard/components/Dialogs/AddExistingSafeDialog/CheckSafe.tsx [hunk 3]
```diff
@@ -51,6 +54,18 @@ const MSG = defineMessages({
...
+  },
+  calloutLink: {
+    id: 'dashboard.AddExistingSafeDialog.CheckSafe.calloutLink',
+    defaultMessage: 'Set up instructions',
+  },
+  warning: {
+    id: 'dashboard.AddExistingSafeDialog.CheckSafe.warning',
+    defaultMessage: 'Warning!',
```
Comment (arrenv) [inline src/modules/dashboard/components/Dialogs/AddExistingSafeDialog/CheckSafe.tsx:67]: This text can change to "Important!" now.

**Review by arrenv — APPROVED** [review]
@ArmandoGraterol Looks good, just a couple of minor fixes.

**Pull request merged**